### PR TITLE
Prepare 3.0 final

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

File: demo/iron-doc-module.html [hunk 1]
```diff
@@ -1,6 +1,4 @@
-<script type="module" src="../default-theme.js"></script>
-<script type="module" src="../iron-doc-module.js"></script>
-<script src="../../../@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2/webcomponents-loader.js"></script>
 
 <!-- FIXME(polymer-modulizer):
```
Comment (keanulee) [inline demo/iron-doc-module.html]: Remove FIXME comment

**Review by keanulee — CHANGES_REQUESTED** [review]

File: demo/iron-doc-module.html (hunk 1 above)
Comment (aomarks) [reply demo/iron-doc-module.html]: Note this is a different FIXME to the one we discussed, it's https://github.com/Polymer/polymer-modulizer/issues/154.

In looking into this, I realized that the existing index.html was a better demo anyway, so I moved that here. That file *also* had the styling FIXME issue, but I found I could restore to the standard form and there were no styling issues. Not sure if that was just in this case, or if something was fixed in ShadyCSS.

**Review by e111077 — CHANGES_REQUESTED** [review]
please delete manifest.json

**Review by e111077 — APPROVED** [review]
nevermind keep it

**Review by dfreedm — COMMENTED** [review]
Travis changes LGTM

**Comment by aomarks** [comment]
@keanulee @e111077 PTAL, updated README.

File: demo/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <script type="module">
+      import '@polymer/polymer/lib/elements/dom-bind.js';
+      import '@polymer/iron-ajax/iron-ajax.js';
+      import '../default-theme.js';
+      import '../iron-doc-viewer.js';
+    </script>
+
+    <custom-style>
+      <style is="custom-style" include="iron-doc-default-theme"></style>
+    </custom-style>
```
Comment (keanulee) [inline demo/index.html:25]: Does this work? From what I tried with `include="demo-pages-shared-styles"` in other elements, this needs to be stamped out in a `<script type="module">`. I made a suggestion regarding this in the doc.

Comment (aomarks) [reply demo/index.html:25]: Yes, it works for me as-is.

**Comment by aomarks** [comment]
Ping @keanulee @e111077

File: default-theme.js [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
+*/
 import '@polymer/paper-styles/color.js';
 import '@polymer/paper-styles/typography.js';
+
 const $_documentContainer = document.createElement('template');
 $_documentContainer.setAttribute('style', 'display: none;');
```
Comment (e111077) [inline default-theme.js:15]: can't you just put this in the dom?

File: iron-doc-viewer-styles.js [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
+*/
+
 const $_documentContainer = document.createElement('template');
 $_documentContainer.setAttribute('style', 'display: none;');
```
Comment (e111077) [inline iron-doc-viewer-styles.js:13]: same with this one. Can you just put the dom-module in the dom?

**Review by e111077 — CHANGES_REQUESTED** [review]

File: default-theme.js (hunk 3 above)
Comment (aomarks) [reply default-theme.js:15]: It's a JS file

**Review by keanulee — APPROVED** [review]

File: default-theme.js (hunk 3 above)
Comment (e111077) [reply default-theme.js:15]: helo I'm dum

**Review by e111077 — APPROVED** [review]

**Pull request merged**